packages/webapp/src/components/lists/ReportList/index.tsx

```diff
@@ -86,29 +105,32 @@ export const ReportList: StandardFC<ReportListProps> = wrap(function ReportList(
 					onReportOptionChange={handleReportTypeChange}
 					onShowFieldsChange={handleShowFieldsChange}
 					onExportDataButtonClick={downloadCSV}
+					onPrintButtonClick={handlePrint}
 					numRows={filteredData.length}
 					unfilteredData={unfilteredData}
 					selectedService={selectedService}
 					hiddenFields={hiddenFields}
 				/>
-				<Report
-					type={reportType}
-					data={filteredData}
-					service={selectedService}
-					hiddenFields={hiddenFields}
-					setFilteredData={setFilteredData}
-					setUnfilteredData={setUnfilteredData}
-					setCsvFields={setCsvFields}
-					{...{
-						filterColumns,
-						filterColumnTextValue,
-						filterRangedValues,
-						getDemographicValue,
-						fieldFilters,
-						setFieldFilters,
-						setFilterHelper
-					}}
-				/>
+				<div id='reportList'>
```

Comment:
Why do you need this `<div id="reportList">` here?

Reply:
oops good catch, this was used during testing while trying different methods of printing, its been removed